conn.go

```diff
@@ -117,13 +117,13 @@ func createConn(nextConn net.Conn, flightHandler flightHandler, handshakeMessage
 			if err != nil {
 				c.stopWithError(err)
 				return
-			} else if c.getConnErr() != nil {
+			} /*else if c.getConnErr() != nil {
```

Comment:
Same as below, mind keeping this PR just to string/moving files around. We had a big move like this in pions/webrtc once and really burned me.